src/core/css/ErrorHandler.scss

```diff
@@ -0,0 +1,15 @@
+@import "~core/css/inc/vars";
```

Comment:
I'm confused about this file. Did you mean to delete some code from [ui/components/ErrorList/styles.scss](https://github.com/mozilla/addons-frontend/blob/86ac2fe2541df5dbcb8038deb7af609aa06192bd/src/ui/components/ErrorList/styles.scss) too?